content/zh/docs/installing-on-linux/introduction/multioverview.md

```diff
@@ -223,24 +211,24 @@ spec:
     port: "6443"
 ```
 
-#### hosts
+#### 主机
 
-请在 `hosts` 下列出您所有安装机器的详细信息。
+参照上方示例在 `hosts` 下列出您的所有机器并添加详细信息。
 
-`name`：实例的主机名称。
+`name`：实例的主机名。
 
-`address`：此 IP 地址用于您通过 SSH 从任务执行机连接至其他实例，可以是公共 IP 或私有 IP 地址，具体取决于安装环境。例如，部分云平台会为每个实例提供一个公共 IP 地址，用于通过 SSH 进行访问。在这种情况下，请在此字段填入该公共 IP 地址。
+`address`：任务机和其他实例通过 SSH 相互连接所使用的 IP 地址。根据您的环境，可以是公共 IP 地址或私有 IP 地址。例如，一些云平台为每个实例提供一个公共 IP 地址，用于通过 SSH 访问。在这种情况下，您可以在该字段填入这个公共 IP 地址。
 
 `internalAddress`：实例的私有 IP 地址。
 
-- 本教程中端口 22 是 SSH 的默认端口。如果您使用其他端口，请在 IP 地址后添加对应端口号，例如：
+- 在本教程中，端口 22 是 SSH 的默认端口，因此您无需将它添加至 Yaml 文件中。否则，您需要在 IP 地址后添加对应端口号。例如：
```

Comment:
Yaml -> YAML

Please change the English version to YAML as well.

Reply:
Just to confirm with you that YAML always appears with each letter capitalized?

Reply:
No. If you use yaml to indicate a specific file, use lowercase letters, such as `config-sample.yaml`. If you use YAML individually to represent the format, use uppercase letters.

Reply:
Got it.